src/main/java/com/crowdin/client/projectsgroups/model/Project.java

```diff
@@ -33,5 +33,7 @@ public class Project {
     private Language sourceLanguage;
     private List<Language> targetLanguages;
     private String webUrl;
+    private Integer defaultTmId;
+    private Integer defaultGlossaryId;
```

Comment:
Add it to the [`ProjectSettings.java`](https://github.com/crowdin/crowdin-api-client-java/blob/master/src/main/java/com/crowdin/client/projectsgroups/model/ProjectSettings.java) model instead of Project